data/campaigns/World_Conquest/lua/game_mechanics/effects.lua

```diff
@@ -91,7 +91,7 @@ end
 -- Like apply_to=defense with replace=true, but never decreases defense.
 function wesnoth.effects.wc2_min_defense(u, cfg)
 	local defense_new = {}
-	local defense_old = wml.parsed(wml.get_child(cfg, "defense"))
+	local defense_old = wml.get_child(cfg, "defense")
```

Comment:
Here too I think the `wml.parsed` should be left in, for the same reason.

Reply:
I have restored the `wml.parsed` here, in the second commit.